lib/core/src/server/build-dev.js

```diff
@@ -151,12 +183,46 @@ Would you like to run Storybook on port ${port} instead?`,
   });
 
   try {
-    const [stats] = await Promise.all([webpackValid, serverListening]);
+    const [stats, updateInfo] = await Promise.all([webpackValid, updateCheck(), serverListening]);
     const proto = program.https ? 'https' : 'http';
     const address = `${proto}://${program.host || 'localhost'}:${port}/`;
     const networkAddress = `${proto}://${ip.address()}:${port}/`;
-    logger.info(`Storybook started on => ${chalk.cyan(address)}`);
-    logger.info(`Available on the network at => ${chalk.cyan(networkAddress)}\n`);
+
+    const updateMessage =
+      updateInfo.success && semver.lt(packageJson.version, updateInfo.data.latest.version)
+        ? stripIndents`
+          ${chalk.hex('#F3AD38')(
+            `A new version (${chalk.bold(updateInfo.data.latest.version)}) is available!`
+          )}
+          ${chalk.gray(updateInfo.data.latest.info.plain)}
+
+          ${chalk.gray('Read full changelog here:')}
+          ${chalk.gray.underline('https://git.io/fxc61')}
+        `
+        : '';
+
+    const serveMessage = stripIndents(
+      Table([['Local', chalk.cyan(address)], ['On your network', chalk.cyan(networkAddress)]], {
+        borderStyle: 0,
+        paddingLeft: 0,
+        compact: true,
+        paddingBottom: 0,
+        align: 'left',
+        color: 'white',
+      }).render()
+    );
+
+    console.log(
```

Comment:
Should this not be `logger.log`?

Reply:
In the screenshot the description is hard to read, but that's how I have configured my terminal profile.

It's using 'gray' so by default it will be visible.